tests/main/hidden-snap-dir/task.yaml

```
@@ -184,3 +184,23 @@ execute: |
     check_env "--with-exposed-home" "x3"
     check_dirs "--with-exposed-home" "x3"
     check_data "--with-exposed-home" "x3" "$data"
+
+    echo "Check fresh install of core22 based snap"
+    snap remove --purge "$NAME"
+    rm -rf "$HOME"/Snap "$HOME"/.snap/data "$HOME/snap/$NAME"
+
+    snap install --dangerous "$NAME"_1.0_all.snap
+
+    check_env "--with-exposed-home" "x1"
```

this should work too:
```suggestion
    check_env --with-exposed-home x1
```